bots/make-checkout

```diff
@@ -46,54 +48,36 @@ def main():
 
     api = github.GitHub(repo=opts.repo)
 
-    def execute(*args):
+    def execute(*args, cwd=BASE, error_on_fail=True):
```

Comment:
ha, had no idea this can be done (could not be done in python2) :)
https://docs.python.org/3/glossary.html#keyword-only-parameter

Reply:
I think I was surprised as well, I just tried it out and it seemed to work. Had no idea it wasn't a thing in python2 lol.